robots/commenter/main.go

```diff
@@ -257,3 +308,45 @@ func run(c client, query, sort string, asc, random bool, commenter func(meta) (s
 	}
 	return nil
 }
+
+func getLastValidLabelsUpdate(ctx context.Context, client *github2.Client, issue github.Issue, goodLabels map[string]bool, badLabels map[string]bool) (time.Time, error) {
+	org, repo, number, err := parseHTMLURL(issue.HTMLURL)
+	if err != nil {
+		return time.Time{}, fmt.Errorf("error parsing HTML of PR %d: %v", issue.ID, err)
+	}
+	lastLabelDate := time.Time{}
+	for i := 1; true; i++ {
+		issueTimeline, _, err := client.Issues.ListIssueTimeline(ctx, org, repo, number, &github2.ListOptions{PerPage: 100, Page: i})
+		if err != nil {
+			return time.Time{}, fmt.Errorf("could not get PR timeline PR %d in repo %s/%s: %v", issue.ID, org, repo, err)
+		}
+		// break loop if there is no timeline entry in PR
+		if len(issueTimeline) == 0 {
+			break
+		}
+		// loop all events, search for labels and check the create date for good labels
+		for _, v := range issueTimeline {
```

Comment:
v is var name is pretty confusing to me. It's someting like a timelineItem, right?